edenia/antelope-tools
# Feat/filter evm rpc endpoints 1288

### Improve UX in endpoints pages

### What does this PR do?

- Resolve #1288
- Filter evm rpc endpoints by default and fix the sorting
- Remove button to show endpoints list, replace it with the copy button to avoid an extra step to get the list
- Use a separate query to get the producer count instead of querying the same data with query and subscription
- Use the same query to get the list of endpoints

### Steps to test

1. Run the project locally
2. Go to endpoints page
3. Write down a search criteria
4. Check the results and click in Copy button
5. Validate the text on the clipboard
6. Go to evm endpoints page
7. Check the filter

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/8af08817-ec96-4cb8-a679-0041adb1826a)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/98348ed0-b638-4276-8f1a-40a64e876a7a)

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/cdea7d6d-ca85-4a66-9190-975d93939059)
